src/progly/run-query.cc

```diff
@@ -292,7 +299,7 @@ int main(int argc, char **argv)
     assert(!projection); // not supported
     std::cout << "select * from lineitem" << std::endl;
 
-  } else if (query == "flatbuf") {
+  } else if (query == "flatbuf" || query == "arrow") {
```

Comment:
I think this check is no longer needed, query defaults to "flatbuf" so no need to specify "arrow".

Reply:
Done